index.js

```diff
@@ -236,17 +236,22 @@ function getter(options) {
 				// since Google doesn't put there '.svg' suffix for some reason
 				// (but does that for other extensions).
 				var re = new RegExp([
-					"\\s*font-family:\\s*'([^']+)';",
-					"\\s*font-style:\\s*(\\w+);",
-					"\\s*font-weight:\\s*(\\w+);",
-					"\\s*src:([^;]*);",
-					".*(?:unicode-range:([^;]+);)?",
+					"\\s*font-family:\\s*'(?<family>[^']+)';",
+					"\\s*font-style:\\s*(?<style>\\w+);",
+					"\\s*font-weight:\\s*(?<weight>\\w+);",
+					".*(?:\\s*font-stretch:\\s(?<stretch>[^;]+);)?",
```

Comment:
Is the .* really needed?
It would actually be better if this entire "we match everything at once" would go away and we would match the stuff we really need instead.

Reply:
A proper parser CSS parser might be useful instead of using Regexes here.  I'm open to PRs.

Reply:
The .* is cpoied from the pattern of `unicode-range`. It could help when there are lines between `font-weight` and `font-stretch`. I agree that a proper CSS parser would be helpful. This PR is only a quick fix.